src/libs/Navigation/Navigation.js

```diff
@@ -46,6 +47,9 @@ function canNavigate(methodName, params = {}) {
 // Re-exporting the getTopmostReportId here to fill in default value for state. The getTopmostReportId isn't defined in this file to avoid cyclic dependencies.
 const getTopmostReportId = (state = navigationRef.getState()) => originalGetTopmostReportId(state);
 
+// Re-exporting the getTopMostCentralPaneRouteName here to fill in default value for state. The getTopMostCentralPaneRouteName isn't defined in this file to avoid cyclic dependencies.
+const getTopMostCentralPaneRouteName = (state = navigationRef.getState()) => originalGetTopMostCentralPaneRouteName(state);
```

Comment:
In almost all cases we just want to get it from the root state. But there is an exception in linkTo.js where we are using the state generated by getStateFromPath. It's a situation similar to the getTopMostReportId

Reply:
So we don't have to import `navigationRef` to call `getState()` and put it in `getTopMostCentralPaneRouteName` everywhere except one case

Reply:
ah so most of the cases, we just call this function without any argument because the function needs the root state which is provided as a default arg.

In specific situations, like within linkTo.js, the function requires a state returned by a different function.

Did I catch that correctly?